src/software/physics/euclidean_to_wheel_test.cpp

```diff
@@ -0,0 +1,127 @@
+#include "software/physics/euclidean_to_wheel.h"
+
+#include <gtest/gtest.h>
+
+#include "software/test_util/test_util.h"
+
+class EuclideanToWheelTest : public ::testing::Test
+{
+   protected:
+    EuclideanToWheelTest() = default;
+    WheelSpace_t current_wheel_speeds{};
+    EuclideanSpace_t target_euclidean_velocity{};
+    WheelSpace_t expected_wheel_speeds{};
+
+    static void SetUp() override
+    {
+        auto euclideanToFourWheel = new EuclideanToWheel(200);
+    }
+
+    static void TearDown() override
+    {
+        delete euclideanToFourWheel;
+    }
```

Comment:
do this and replace euclideanToFourWheel with euclidean_to_four_wheel in this file to fix the build error
```suggestion
    EuclideanToWheel euclidean_to_four_wheel = EuclideanToWheel(200.0f);
```